plugins/auxiliary/rce/SBD_CommandChainer@OWTF-ARCE-003.py

```diff
@@ -9,62 +9,50 @@
 
 def run(PluginInfo):
     # ServiceLocator.get_component("config").Show()
```

Comment:
Forgot the commented code + you have a duplicate import `from framework.utils import OWTFLogger`